app/pb/account.proto

```diff
@@ -22,6 +25,18 @@ service Account {
   }
 }
 
+message GetAccountInfoRes {
+  enum LoginMethod {
+    MAGIC_LINK = 0;
+    PASSWORD = 1;
+  }
+
+  LoginMethod login_method = 1;
```

Comment:
Can't an account have both login methods enabled?

Reply:
I don't think it's a good idea security-wise. A user would presumably either want link or password login, an attacker would want the weakest :P